src/regress_stack/__main__.py

```diff
@@ -104,18 +104,24 @@ def test(concurrency: int):
                 configure(tempest_conf)
         includes_regexes = getattr(mod.module, "TEST_INCLUDE_REGEXES", [])
         exclude_regexes = getattr(mod.module, "TEST_EXCLUDE_REGEXES", [])
-        if not includes_regexes:
-            # If no include defined, it would get too much tests
-            continue
         test_regexes.append((includes_regexes, exclude_regexes))
 
+    test_regexes.append(
+        (
+            os.environ.get("TEST_INCLUDE_REGEXES", []).split("|"),
+            os.environ.get("TEST_EXCLUDE_REGEXES", []).split("|"),
```

Comment:
Split will fail if the environment var is not defined.

```suggestion
            os.environ.get("TEST_INCLUDE_REGEXES", "").split("|"),
            os.environ.get("TEST_EXCLUDE_REGEXES", "").split("|"),
```